### PR TITLE
Bring back babel for ES5 builds

### LINKED ISSUE #3722: Lovelace frontend not loading on Safari iOS 9.3.5
**Home Assistant release with the issue:**
Home Assistant 0.95.4.

**Last working Home Assistant release (if known):**
Never - not even Home Assistant 0.98.5.

**UI (States or Lovelace UI?):**
Lovelace UI

**Browser and Operating System:**
safari on iPad iOS 9.3.5.

**Description of problem:**
The lovelace UI doesent load on iOS 9.x. It works fine on devices with iOS 10.x and later.

**Steps to reproduce:**
1) Open your lovelace dashboard on an iPad with iOS 9.3.5.
2) Authentication works and redirects to the correct view.
3) Here the header loads succesfully, but not the views/cards.

[Screenshot here](https://drive.google.com/file/d/1kFGfjWQ5dMtQ4DYh-CTEqGGS7uHFEh5c/view?usp=sharing)

**Javascript errors shown in the web inspector (if applicable):**
```

```

**Additional information:**
I have written about this in multiple Facebook groups and other users seem to experience this too.

### LINKED ISSUE #3781: gui remains initializing in kiosk browser
**Home Assistant release with the issue:**
0.99.0 and 0.99.2

**Last working Home Assistant release (if known):**
0.98.?

**UI (States or Lovelace UI?):**
lovelace, but states give the same result

**Browser and Operating System:**
* android version: 6.0.1
* browser: A lightweight browser that relies on the System Webview, for low-end devices, also known as Jelly - comes with LineageOS 13.0
* kiosk app: wallpanel (which I suppose uses the Jelly browser)
* Note: google chrome works fine...

**Description of problem:**

The kiosk app does shows "Initializing" remains in that state

The default browser only shows the blue navigation bar without any info

**Javascript errors shown in the web inspector (if applicable):**
don't know how to get it :(

**Additional information:**
Similar issue is reported here: https://community.home-assistant.io/t/problem-with-0-99-and-fully-kiosk-fire-tablets

### PR DESCRIPTION
**Pull request by bramkragten**
Fixes https://github.com/home-assistant/home-assistant-polymer/issues/3781
Fixes https://github.com/home-assistant/home-assistant-polymer/issues/3722

### CODE REVIEW

File: gallery/webpack.config.js [hunk 1]
```diff
@@ -1,49 +1,61 @@
...
+    test: /\.ts$/,
+    use: [
+      {
+        loader: "ts-loader",
+        options: {
+          compilerOptions: latestBuild
+            ? { noEmit: false }
+            : {
+                target: "es5",
+                checkJs: false,
```
Comment (balloob) [inline gallery/webpack.config.js]: is this still necessary?

**Review by balloob — APPROVED** [review]
Going ahead and merging it. Will test a build locally and push hotfix.

File: gallery/webpack.config.js (hunk 1 above)
Comment (bramkragten) [reply gallery/webpack.config.js]: Nope

**Comment by bramkragten** [comment]
hassio is throwing a bunch of TS errors

**Comment by whattheschnell** [comment]
Is this only in dev?

**Comment by bramkragten** [comment]
No, this is for our production builds.

**Pull request merged**